lib/pixelz/api_client/image.rb

```diff
@@ -12,24 +12,40 @@ def post
         Hash.from_xml(res)
       end
 
+      def reject(image_ticket, comment='')
+        res = RestClient.put(BASE_URI + "/Image/Reject/#{image_ticket}", reject_image_payload(comment), headers)
```

Comment:
The line is long, should make it new line. If you use sublime, there is ruler option below, make it 80. If vim, need to ask Paul.

http://stackoverflow.com/questions/25900954/80-characters-right-margin-line-in-sublime-text-3